tests/test_complex_application.py

```diff
@@ -13,7 +13,7 @@
 import pathlib
 from time import sleep
 
-import pandas as pd
+import modin.pandas as pd
```

Comment:
Same here, we should keep tests in pandas, and eventually add more tests in modin.